TASK/TaskVehicleTempAction.md

```diff
@@ -21,22 +21,24 @@ void TASK_VEHICLE_TEMP_ACTION(Ped driver, Vehicle vehicle, int action, int time)
 '11 - turn right + restore wheel pos to center in the end
 '13 - turn left + go reverse
 '14 - turn left + go reverse
-'16 - crash the game after like 2 seconds :)
-'17 - keep actual state, game crashed after few tries
-'18 - game crash
-'19 - strong brake + turn left/right
-'20 - weak brake + turn left then turn right
-'21 - weak brake + turn right then turn left
-'22 - brake + reverse
-'23 - accelerate fast
-'24 - brake
-'25 - brake turning left then when almost stopping it turns left more
-'26 - brake turning right then when almost stopping it turns right more
-'27 - brake until car stop or until time ends
-'28 - brake + strong reverse acceleration
+'16 - plane fly up (crashes game if not in plane) :)
+'17 - plane fly straight (crashes game if not in plane)
+'18 - plane sharp left (crashes game if not in plane)
+'19 - plane sharp right (crashes game if not in plane)
+'20 - strong brake + turn left/right
+'21 - weak brake + turn left then turn right
+'22 - weak brake + turn right then turn left
+'23 - brake + reverse
+'24 - accelerate fast
+'25 - brake
+'26 - brake turning left then when almost stopping it turns left more
+'27 - brake turning right then when almost stopping it turns right more
+'28 - brake until car stop or until time ends
+'29 - brake + strong reverse acceleration
 '30 - performs a burnout (brake until stop + brake and accelerate)
 '31 - accelerate + handbrake
 '32 - accelerate very strong
+'33 - Surface in submarine
```

Comment:
Hey, this is actually an enum, why not actually remove all this from the native description and declare an enum like so?

```c
enum eTempAction {
    TA_NONE = 0,
    TA_WAIT = 1,
    TA_UNUSED = 2,
    TA_BRAKE_REVERSE = 3,
    TA_HANDBRAKE_TURN_LEFT = 4,
    TA_HANDBRAKE_TURN_RIGHT = 5,
    TA_HANDBRAKE_UNTIL_TIME_ENDS = 6,
    TA_TURN_LEFT = 7,
    TA_TURN_RIGHT = 8,
    TA_ACCELERATE = 9,
    TA_TURN_LEFT = 10,
    TA_TURN_RIGHT = 11,
    TA_UNUSED_12 = 12,
    TA_TURN_LEFT_GO_REVERSE = 13,
    TA_TURN_RIGHT_GO_REVERSE = 14,
    TA_PLANE_FLY_UP = 15, // (crashes game if not in plane)
    TA_PLANE_FLY_STRAIGHT = 16, // (crashes game if not in plane)
    TA_PLANE_SHARP_LEFT = 17, // (crashes game if not in plane)
    TA_PLANE_SHARP_RIGHT = 18, // (crashes game if not in plane)
    TA_STRONG_BRAKE = 19,
    TA_TURN_LEFT_AND_STOP = 20,
    TA_TURN_RIGHT_AND_STOP = 21,
    TA_GO_IN_REVERSE = 22,
    TA_ACCELERATE_FAST = 23,
    TA_BRAKE_ACTION = 24,
    TA_HANDBRAKE_TURN_LEFT_MORE = 25,
    TA_HANDBRAKE_TURN_RIGHT_MORE = 26,
    TA_HANDBRAKE_BRAKE_STRAIGHT = 27,
    TA_BRAKE_STRONG_REVERSE_ACCELERATION = 28,
    TA_UNUSED_29 = 29,
    TA_PERFORMS_BURNOUT = 30,
    TA_REV_ENGINE = 31,
    TA_ACCELERATE_VERY_STRONG = 32,
    TA_SURFACE_IN_SUBMARINE = 33
};
```

Reply:
Additionally, a bit worrying the game crashes if the player is not in the vehicle (plane).

Reply:
Beautiful! I'll get cracking on it once I get off my shift! 

Thanks!

Reply:
Instead of all this, can you set a table method? 

| Code | Action                                    |
|------|-------------------------------------------|
| 1    | brake                                     |
| 3    | brake + reverse                           |
| 4    | turn left 90 + braking                    |
| 5    | turn right 90 + braking                   |
| 6    | brake strong (handbrake?) until time ends |
| 7    | turn left + accelerate                    |
| 7    | turn right + accelerate                   |
| 9    | weak acceleration                         |
| 10   | turn left + restore wheel pos to center   |
| 11   | turn right + restore wheel pos to center  |
| 13   | turn left + go reverse                    |
| 14   | turn left + go reverse                    |
| 16   | plane fly up (crashes game if not in plane) |
| 17   | plane fly straight (crashes game if not in plane) |
| 18   | plane sharp left (crashes game if not in plane) |
| 19   | plane sharp right (crashes game if not in plane) |
| 20   | strong brake + turn left/right           |
| 21   | weak brake + turn left then turn right    |
| 22   | weak brake + turn right then turn left    |
| 23   | brake + reverse                           |
| 24   | accelerate fast                           |
| 25   | brake                                     |
| 26   | brake turning left then turn left more    |
| 27   | brake turning right then turn right more  |
| 28   | brake until car stop or until time ends   |
| 29   | brake + strong reverse acceleration      |
| 30   | performs a burnout (brake until stop + brake and accelerate) |
| 31   | accelerate + handbrake                    |
| 32   | accelerate very strong                    |
| 33   | Surface in submarine                      |

Reply:
Also if you can add param descriptions would be great!